test/ClangImporter/experimental_diagnostics_incomplete_types_negative.swift

```diff
@@ -0,0 +1,74 @@
+// RUN: not %target-swift-frontend(mock-sdk: %clang-importer-sdk) -enable-experimental-clang-importer-diagnostics -enable-objc-interop -typecheck %s 2>&1 | %FileCheck %s --strict-whitespace
```

Comment:
Is this really a helpful test? I don't think negative tests are particularly useful. The potential scope is infinite. Also the rest of the test suite will act as a negative test.

Reply:
I don't feel very strongly about it, but I do think it is a somewhat helpful sanity check. I added it because I was close to inadvertently misdiagnosing a complete protocol as incomplete. I'll remove it if you like, but the rest of the test suite is not run with the enabling flag on.

Reply:
> but the rest of the test suite is not run with the enabling flag on.

Good point. Keep it.